src/resources/postcss/widgets/full/_events-list.pcss

```
@@ -88,11 +88,17 @@
 	.site-footer .widget-area & {
 
 		.tribe-events-widget-events-list__view-more-link {
-			color: var(--tec-color-accent-primary);
+			color: var(--tec-color-link-accent);
 
-			&:hover,
-			&:focus {
-				color: var(--tec-color-accent-primary);
+			&:visited {
+				color: var(--tec-color-link-accent);
+			}
```

And remove the above:

```suggestion
			&,
			&:visited {
				color: var(--tec-color-link-accent);
			}
```